content/podcast/s03e06-brave/index.md

```diff
@@ -0,0 +1,64 @@
+draft = false
+aliases = ["/p/s03e07"]
+[extra]
+guest = "Anton Lazarev"
+role = "Senior Software Engineer"
+season = "03"
+episode = "06"
+series = "Podcast"
++++
+
+Web browsers today face increasing demands for both performance and privacy. At Brave, they're tackling both challenges head-on with their Rust-based ad-blocking engine. This isn't just about blocking ads – it's about doing so with minimal performance impact while maintaining compatibility with existing filter lists and adapting to evolving web technologies.
+
+<!-- more -->
+
+Today we're joined by Anton Lazarev, Senior Software Engineer at Brave, to discuss how they're using Rust to build a high-performance ad-blocker, the challenges of working within browser constraints, and how they're pushing the boundaries of what's possible in modern web privacy.
+
+## Show Notes
+
+### About Brave
+
+Brave is a privacy-focused web browser that blocks ads and trackers by default. It's built on top of Chromium, with additional privacy features like the ability to earn cryptocurrency by viewing privacy-respecting ads. Brave has been an early adopter of Rust, using it to build their ad-blocking engine and other core features.
+
+### About Anton Lazarev
+
+Anton Lazarev is a Senior Software Engineer at Brave, where he works on the browser's core privacy features. 
+In his free time, Anton enjoys working on like his fully open-source and Rust-based, customizable split keyboard.
+
+### Links From The Episode (In Chronological Order)
+
+- [VST](https://en.wikipedia.org/wiki/Virtual_Studio_Technology) - Audio plugin standard
+- [Azul](https://azul.rs/) - Early Rust UI library
+- [Rust subreddit "Who's Hiring"](https://www.reddit.com/r/rust/search?q=title%3A%22Who%27s+Hiring%22&restrict_sr=on&sort=new) - Looking for a Rust job?
+- [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) - Alternative to CSS style selectors
+- [CXX](https://cxx.rs/) - C/C++ Rust Interop crate
+- [CNAME cloaking](https://brave.com/privacy-updates/6-cname-trickery/) - Technique for disguising tracking domains
+- [Manifest V3](https://developer.chrome.com/docs/extensions/mv3/intro/) - New extension platform specification
+- [uBlock Origin](https://github.com/gorhill/uBlock) - Popular content blocker
+- [adblock-rust](https://github.com/brave/adblock-rust) - Brave's adblock engine written in Rust
+- [Public Suffix List](https://publicsuffix.org/) - Explains what a "suffix" is compared to a TLD
+- [Servo2](https://servo.org/) - Next generation browser engine
+- [Ladybird](https://ladybird.org/) - Browser project
+- [Swift supported compile targets](https://github.com/swiftlang/swift/blob/131e504c16f069305c56fd25b8d6373639a6f32a/utils/swift_build_support/swift_build_support/targets.py#L347-L427)
+- ["The Dying Web" by Matthias](https://endler.dev/2024/the-dying-web/)
+- [GN](https://gn.googlesource.com/gn/) - Old Chromium build system
+- [GNRT](https://example.com/gnrt) - New Chromium build system
```

Comment:
I don't think this will work :sweat_smile:

Reply:
What exactly?

Reply:
Fixed (hopefully)

Reply:
Nothing is fixed?!

Reply:
Look again. 👀 🪄 😆